lib/field_names.cpp

```diff
@@ -48,15 +48,22 @@ FieldNames::init(const ResultBase* res)
 }
 
 
+#if __cplusplus >= 201103L
+unsigned int
+FieldNames::operator [](std::string s) const
+{
+    std::string temp1(std::move(s));
+#else
 unsigned int
 FieldNames::operator [](const std::string& s) const
 {
 	std::string temp1(s);
+#endif
```

Comment:
Again, you don't get to change the method signature like this. It's an unjustifiable ABI breakage.